paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/TransformSpecToElement.kt

```diff
@@ -75,6 +54,32 @@ private fun transform(
     )
 }
 
+private fun transform(
```

Comment:
Thinking out loud: Does this make sense as an extension function on `List<SectionFieldSpec>`?

Reply:
I tried this out and I really liked it.  It helps scope it to just a specific class instead of it being global.